apps/demo/src/locales/en/ui.json

```
@@ -8,5 +8,8 @@
     "export": "Export XLSX",
     "submit": "Submit Button"
   },
-  "responsiveMenu": "Responsive Menu"
+  "responsiveMenu": "Responsive Menu",
+  "files":{
+    "table":"Files Table"
+  }
```

Follow sentence case for all literals
```suggestion
  "responsiveMenu": "Responsive menu",
  "files":{
    "table":"Files table"
  }
```